File: 2019/FastestFizzbuzz/fizzbuzz.py
#!/usr/bin/env python3

def main():
    for i in range(1, 1000000001):
        fizz = (i % 3 == 0);
        buzz = (i % 5 == 0);
        if fizz and buzz:
            print("Fizzbuzz!")
        elif fizz:
            print("Fizz")
        elif buzz:
            print("Buzz")
        else:
            print("{:010d}".format(i))

if __name__ == "__main__":
    main()

File: 2019/FastestFizzbuzz/fizzbuzz.js
for(var i=1; i<=1000000000; i++){
    let fizz = (i%3 == 0);
    let buzz = (i%5 == 0);
    if(fizz && buzz){
        console.log("Fizzbuzz!");
    }
    else if(fizz){
        console.log("Fizz");
    }
    else if(buzz){
        console.log("Buzz");
    }
    else{
        let s = i.toString()
        let pad = "0".repeat(10-s.length)
        console.log(pad+s)
    }
}
